docker/daemon_linux.go

```diff
@@ -0,0 +1,30 @@
+// +build daemon,linux
```

Comment:
This file should be called `daemon_unix.go` and use the build tag `!windows` rather than `linux`.